wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/CommandBase.java

```diff
@@ -29,7 +30,13 @@ protected CommandBase() {
    * @param requirements the requirements to add
    */
   public final void addRequirements(Subsystem... requirements) {
-    m_requirements.addAll(Set.of(requirements));
+    for (Subsystem requirement : requirements) {
+      if (requirement == null) {
+        DriverStation.reportWarning("Tried to add a null requirement", true);
+        continue;
```

Comment:
I'm a bit wary about this; since this is a very common occurrence then maybe we should throw here instead of warning and ignoring?

Reply:
I'd say you should throw here; if this is happening dynamically in user code then the user is doing something very complicated and on their head be it.